phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexUpgradeTool.java

```diff
@@ -648,17 +648,18 @@ private boolean extractTablesAndIndexes(PhoenixConnection conn) {
         }
 
         if (hasViewIndex) {
-            String viewSql = "SELECT DISTINCT TABLE_NAME, TENANT_ID FROM "
-                    + "SYSTEM.CATALOG "
-                    + "WHERE COLUMN_FAMILY = \'" + dataTableFullName + "\' "
+            String viewSql = "SELECT DISTINCT COLUMN_FAMILY, TENANT_ID FROM "
```

Comment:
can this be extracted into a reusable function that verifyViewAndViewIndexes can also use? Right now the test is verifying that its _own_ query works as expected, but not the actual upgrade tool's. Imagine if someone changed IndexUpgradeTool:651 and broke it -- the test would still pass.